### ISSUE TITLE
haxelib run openfl does nothing, no error!

### ISSUE THREAD

**Issue by bluelemonade** [HaxeFoundation/haxelib#434]
Hello,

I tried to installe haxe/haxelib on a windows 10 machine. no way! after haxelib install openfl I tried the haxelib run openfl setup --verbose

nothing, no error, no answer? wheres the problem: I have althoug the npm version installed. Is there a problem? network problems? 

regards.

**Comment by EricBishton** [comment]
Check the error code from haxelib immediately after running haxelib  (e.g. "echo %ERRRNO%" on windows, "echo $!" on linux/mac).  If it's a large (possibly negative) number, then it means that the Neko runtime either isn't installed or isn't being found on your path.  (For some unknown reason, Neko isn't bundled with the Haxe runtime even though it's required in order to use haxelib.)  You can get Neko at https://nekovm.org.

**Comment by bluelemonade** [comment]
hi,

with %errorlevel% i get -1073741819.   echo %ERRRNO% returns echo %ERRRNO%

with a simple neko I become: NekoVM 2.1.0 (c)2005-2016 Haxe Foundation
  Usage : neko <file>

**Comment by bluelemonade** [comment]
ok, some problems with UAC, a known windows issue...

**Comment by bluelemonade** [comment]
ok, would be nice to habve the installation on my basic working account. Is there a way to install these things by "hand" one after another. perhaps I can execute a script and see when the error happens.

thanks

**Comment by ibilon** [comment]
If you do `haxelib list` do you see openfl installed?
If yes and since neko seems to work I'm not sure it's a haxelib issue.

**Comment by bluelemonade** [comment]
yes, thats all fine, I can install... uninstall, get warning when although a lib is installed...
only run openfl setup didn't work.

**Comment by RealyUniqueName** [comment]
Doesn't look like we need to act here since it was stated to be an UAC issue.
Is this still relevant?

**Comment by guilhermetack** [comment]
:l
